daemon/daemon_unix.go

```diff
@@ -186,7 +190,10 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
 
 		switch con[0] {
 		case "label":
-			labelOpts = append(labelOpts, con[1])
+			if selinuxConvert {
+				opt = strings.Replace(opt, ":", "=", 1)
```

Comment:
wait, but isn't this converted already above? this would be in a case where `--security-opt label:foo=something`?